src/controllers/controllerengine.cpp

```diff
@@ -681,9 +681,12 @@ void ControllerEngine::setValue(QString group, QString name, double newValue) {
 
     ControlObjectScript* coScript = getControlObjectScript(group, name);
 
-    if (coScript != nullptr) {
-        ControlObject* pControl = ControlObject::getControl(
-                coScript->getKey(), ControlFlag::NoAssertIfMissing);
+    if (coScript) {
+        ControlObject* pControl = ControlObject::getControl(coScript->getKey(),
+                ControllerDebug::enabled()
+                        ? ControlFlag::None
+                        : (ControlFlag::AllowInvalidKey |
```

Comment:
Shall we define a `constexpr` for this common combination of flags instead of repeating it?

We should also encapsulate the `ControllerDebug::enabled()` use case in a short inline function to improve readability and for future reuse.

Reply:
Whoops this slipped in by mistake and should have been part of #3056 anyway.